Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -197,6 +208,20 @@ double AlignMolecule(ROMol &prbMol, const ROMol &refMol, int prbCid = -1,
   return rmsd;
 }
 
+double GetBestRMS(ROMol &prbMol, ROMol &refMol, int prbId = -1,
+                  int refId = -1, int maxMatches = 1000000,
+                  python::object map = python::list()) {
+  std::vector<MatchVectType> aMapVec = _translateAtomMapVector(map);
```

Comment:
The default value of map should be `python::object()` (this corresponds to `None` in Python), there's a reminder about this below.
Then here you would do:
```
std::vector<MatchVectType> aMapVec;
if(map != python::object()) aMapVec = _translateAtomMapVector(map);
```